File: GoLang/6 kyu/Convert string to camel case/main.go
package main

import (
	"fmt"
	"strings"
	"regexp"
)

func arrayToString(words []string) string {
	result := ""
	for index, word := range words {
		if index != 0 {
		  result += strings.Title(word)
		} else {
		  result += word
		}
	  }
	return result
}

func ToCamelCase(s string) string {
	words := regexp.MustCompile("-|_").Split(s, -1)
	camelString := arrayToString(words)
	return camelString
}

func main() {
	fmt.Println(ToCamelCase("the-very_beginning"))
}

File: Python/6 kyu/Convert string to camel case/main.py
def to_camel_case(text):
    res = ""
    arr = []
    text = text.replace("-", "_")
    arr = text.split("_")

    for i in range(len(arr)):
        if i == 0:
            res += arr[i]
        else:
            res += arr[i].capitalize()
    return res

print(to_camel_case("the-very_beginning"))